plugins/woocommerce-blocks/tests/e2e/tests/checkout/additional-fields.shopper.block_theme.spec.ts

```diff
@@ -749,11 +749,11 @@ test.describe( 'Shopper → Additional Checkout Fields', () => {
 
 			// Check text inputs in edit mode match the expected values.
 			const govIdInput = checkoutPageObject.page.getByLabel(
-				'Government ID *',
+				'Government ID',
```

Comment:
```suggestion
				'Government ID *',
```

Due to `{ exact: true }`, removing the asterisk causes this e2e test to fail.